handlers/api_eth1.go

```diff
@@ -973,6 +1009,10 @@ func getValidatorExecutionPerformance(queryIndices []uint64) ([]types.ExecutionP
 			producerReward = mevBribe
 		}
 
+		if block.Time.AsTime().After(firstEpochTime) {
+			result.PerformanceTotal = result.PerformanceTotal.Add(result.PerformanceTotal, producerReward)
+			result.Performance365d = result.Performance365d.Add(result.Performance365d, producerReward)
+		}
 		if block.Time.AsTime().After(last30dTimestamp) {
```

Comment:
Minor: It slightly annoys me that this is called **last30dTimestamp** when it is for 31 days.

Reply:
So it shell be renamed!